src/ui/hexedit.cc

```diff
@@ -183,6 +183,31 @@ HexEdit::HexEdit(FileBlobModel *dataModel, QItemSelectionModel *selectionModel,
         util::settings::shortcuts::REMOVE_CHUNK, this, Qt::WidgetWithChildrenShortcut);
   connect(removeChunkPassiveAction, &QAction::triggered, remove_chunk_lambda);
 
+  selectChunkAction_ = ShortcutsModel::getShortcutsModel()->createQAction(
+        util::settings::shortcuts::SELECT_CHUNK, this, Qt::WidgetWithChildrenShortcut);
+  connect(selectChunkAction_, &QAction::triggered, [this]() {
+    if (!selectedChunk().isValid()) {
+        return;
+    }
+
+    qint64 byteOffset;
+    qint64 size;
+
+    getRangeFromIndex(selectedChunk(), &byteOffset, &size);
+
+    if (size == 0) {
+      return;
+    }
+
```

Comment:
You can remove some empty lines from this lambda, we have definitely too many of them here ;)